app/scicrunch_requests.py

```diff
@@ -203,9 +217,18 @@ def create_filter_request(query, terms, facets, size, start):
 def get_facet_type_map():
     return {
         'species': ['organisms.primary.species.name.aggregate', 'organisms.sample.species.name.aggregate', 'organisms.scaffold.species.name.aggregate'],
+        'Species': ['organisms.primary.species.name.aggregate', 'organisms.sample.species.name.aggregate',
+                    'organisms.scaffold.species.name.aggregate'],
         'gender': ['attributes.subject.sex.value'],
+        'Gender': ['attributes.subject.sex.value'],
+        'sex': ['attributes.subject.sex.value'],
+        'Sex': ['attributes.subject.sex.value'],
```

Comment:
Sorry late reply on this. It's good to have as scicrunch returns lowercase facets, as Algolia returns upercase facets. So it is nice to have both lower and upercase when we are migrating between them right now.

I could have switched all algolia facets to lowercase when we store them, but I thought since we are using Algolia for facets it will be better to slowly switch to what they use natively for facets